packages/curve-ui-kit/src/features/connect-wallet/ui/ConnectWalletPrompt.tsx

```diff
@@ -16,16 +24,62 @@ export const ConnectWalletPrompt = ({
   connectWallet,
   isLoading,
 }: ConnectWalletPromptProps) => (
-  <ConnectWallet
-    connectWallet={connectWallet}
-    description={description}
-    connectText={connectText}
-    loadingText={loadingText}
-    isLoading={isLoading}
-    theme={useUserProfileStore((state) => state.theme)}
-  />
+  <Stack
+    padding={7}
+    sx={{
+      backgroundColor: 'var(--table--background-color)',
+      maxWidth: '50rem',
+    }}
+  >
+    <Box display="flex" position="relative" width="100%">
+      <Box
+        component="img"
+        src={getBackgroundUrl(useUserProfileStore((state) => state.theme))}
+        alt="Curve Logo"
+        width="1400px"
+        maxWidth="100%"
+        height="100%"
+        sx={{ objectFit: 'contain' }}
+      />
+      <Stack
+        position="absolute"
+        alignItems="center"
+        width="100%"
+        top="50%"
+        left="50%"
+        sx={{ transform: 'translate(-50%, -50%)' }}
+        spacing={3}
+      >
+        <CurveLogo src={LogoImg} alt="Curve Logo" />
+        <Typography variant="headingXxl">Enter Curve</Typography>
+      </Stack>
+    </Box>
+    <Stack gap={3} alignItems="center" width="100%" margin="0 auto">
+      <Typography variant="bodyMRegular">{description}</Typography>
+      <Button
+        size="large"
+        color="primary"
+        onClick={connectWallet}
+        loading={isLoading}
+        loadingPosition="start"
+        data-testid={`btn-connect-prompt${isLoading ? '-loading' : ''}`}
```

Comment:
I suppose you wanted to test this with cypress but forgot?

Reply:
it's tested already: 
https://github.com/curvefi/curve-frontend/blob/5f4abc5f69631751ea4eb246f09390698183441e/tests/cypress/e2e/all/header.cy.ts#L160